.github/workflows/test.yml

```diff
@@ -79,41 +126,30 @@ jobs:
       - uses: nttld/setup-ndk@v1
         with:
           ndk-version: r25c
-      - name: Show riscv release directory
-        run: ls -R target/riscv32imc-unknown-none-elf/release/
       - run: sudo apt-get update && sudo apt install -y ninja-build libstdc++-12-dev libgtk-3-0 libgtk-3-dev libudev-dev
-      - name: Cache Flutter dependencies
-        uses: actions/cache@v3
-        with:
-          path: ~/.pub-cache
-          key: ${{ runner.os }}-flutter-${{ hashFiles('**/pubspec.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-flutter-
```

Comment:
why remove this?

Reply:
Oops, added this back and did some more crazy caching for the rust dependencies.